packages/smart-contracts/scripts/deploy-payments.ts

```diff
@@ -10,10 +10,10 @@ import { uniswapV2RouterAddresses, jumpToNonce } from './utils';
 import { Contract } from 'ethers';
 // eslint-disable-next-line
 // @ts-ignore Cannot find module
-import { ChainlinkConversionPath } from '../src/types/ChainlinkConversionPath';
```

Comment:
I had lots of issues due to dependencies (probably because removing some deps changes the resolved version of other packages) and had to upgrade some. This change is related to the typechain upgrade